### DISCUSSION TITLE
target-equivalent to quarto::quarto_render( output_format = "html")

### DISCUSSION THREAD

**Discussion by dmenne** [ropensci/targets#1013] [Help] [answered]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

What is the equivalent to `quarto_render( output_format = "html")`

I Tried

`tar_quarto(report,  execute_params = list(output_format = "html"))`

It does not complain, but builds the other formats anyway

**Comment by wlandau** [comment]
`tarchetypes::tar_quarto()` is indeed the recommended way to do this, but `output_format` is an argument to `tar_quarto()` and gets ignored when it is part of `execute_params`.

**Reply by dmenne** [reply]
Thanks, I noted that it gets ignored, but how do I say: build html only?

**Reply by wlandau** [reply, marked as answer]
You can set an output format in the YAML front matter of the Quarto document or in `_quarto.yml`. `tarchetypes::tar_quarto()` uses `quarto::quart_inspect()` to discover what the output format and file will be.
